File: ProjetSopraFinal/src/main/java/com/example/demo/model/Gestionnaire.java
package com.example.demo.model;

import java.util.Set;

import javax.persistence.DiscriminatorValue;
import javax.persistence.Entity;
import javax.persistence.OneToMany;

import com.example.demo.jsonview.JsonViews;
import com.fasterxml.jackson.annotation.JsonView;

@Entity
@DiscriminatorValue("G")
public class Gestionnaire extends RessourceHumaine {

	private Set<Formation> formations;

	public Gestionnaire() {

	}

	@JsonView(JsonViews.Common.class)
	@OneToMany(mappedBy = "gestionnaire")
	public Set<Formation> getFormations() {
		return formations;
	}

	public void setFormations(Set<Formation> formations) {
		this.formations = formations;
	}
	
	
}

File: AngularSopraFinal/src/app/model/gestionnaire.ts
import {Formation} from './formation';
import {RessourceHumaine} from './ressourceHumaine';

export class Gestionnaire extends RessourceHumaine{
  constructor(private _formations?) {
    super('stagiaire');
  }

  public get fromations(): Array<Formation> {
    return this._formations
  }
  public set formations(_formations: Array<Formation>) {
    this._formations = _formations;
  }
}